app/res/layout/first_start_screen.xml

```diff
@@ -44,7 +44,7 @@
                <LinearLayout
                    android:id="@+id/message_panel"
                    android:layout_width="fill_parent"
-                   android:layout_height="wrap_content"
+                   android:layout_height="match_parent"
```

Comment:
this doesn't impact the look/feel for the most part, right? Only if you use markdown?